src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/AbstractRequest.java

```diff
@@ -25,21 +26,21 @@
 {
 
     @NotNull
-    private final IToken            token;
+    private final IToken       token;
     @NotNull
-    private final R                 requested;
+    private final R            requested;
     @NotNull
-    private final ArrayList<IToken> children;
+    private final List<IToken> children;
     @NotNull
-    private final IRequester        requester;
+    private final IRequester   requester;
     @NotNull
     private RequestState state = RequestState.CREATED;
     @Nullable
     private R      result;
     @Nullable
     private IToken parent;
     @NotNull
-    private final ItemStack deliveryStack = ItemStackUtils.EMPTY;
+    private final static ItemStack deliveryStack = ItemStackUtils.EMPTY;
```

Comment:
NO this should not be static! We have mutliple different delivery stacks, so making this static defeats ist purpose.